frontend/src/properties/propertysections/WidgetContainerPropertySection.tsx

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import Grid, { GridDirection } from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import { WidgetContainerLayout } from 'form/Form.types';
+import { widgetToPropertySection } from 'properties/PropertySectionOperations';
+import { PropertySectionLabel } from 'properties/propertysections/PropertySectionLabel';
+import { WidgetContainerPropertySectionProps } from 'properties/propertysections/WidgetContainerPropertySection.types';
+import React from 'react';
+
+const useWidgetContainerPropertySectionStyles = makeStyles(() => ({
+  root: {
+    flexGrow: 1,
+  },
+}));
+
+const getLayout = (layout: WidgetContainerLayout): GridDirection => {
+  if (layout === WidgetContainerLayout.HORIZONTAL) {
+    return 'row';
+  }
+  return 'column';
+};
+
+export const WidgetContainerPropertySection = ({
+  editingContextId,
+  formId,
+  widget,
+  widgetSubscriptions,
+  setSelection,
+  readOnly,
+}: WidgetContainerPropertySectionProps) => {
+  const classes = useWidgetContainerPropertySectionStyles();
+
+  const layout = getLayout(widget.layout);
+
+  return (
+    <div>
+      <PropertySectionLabel label={widget.label} subscribers={[]} />
+      <Grid
+        className={classes.root}
+        data-testid={widget.label}
+        container
+        spacing={2}
```

Comment:
We don't want to set default values in the frontend. If you really need to set a default value, make it available in the view DSL and set the default there.